shell/platform/darwin/ios/framework/Source/FlutterChannelKeyResponder.mm

```diff
@@ -139,7 +135,7 @@ - (void)handlePress:(nonnull FlutterUIPressProxy*)press
   NSString* characters = getEventCharacters(press.key.characters, press.key.keyCode);
   NSString* charactersIgnoringModifiers =
       getEventCharacters(press.key.charactersIgnoringModifiers, press.key.keyCode);
-  NSMutableDictionary* keyMessage = [[@{
+  NSMutableDictionary* keyMessage = [@{
```

Comment:
You can just make this an `NSDictionary` and remove the `mutableCopy`; `sendMessage:` doesn't require a mutable dictionary. I looked at the history; there used to be conditional additional key/value setting logic, which is why it was mutable, but now that's just cruft.

Reply:
Done.